src/NUnitEngine/nunit.engine.core/Services/ExtensionService.cs

```diff
@@ -41,14 +41,16 @@ namespace NUnit.Engine.Services
     public class ExtensionService : Service, IExtensionService
     {
         static Logger log = InternalTrace.GetLogger(typeof(ExtensionService));
-        static readonly Version ENGINE_VERSION = typeof(TestEngine).Assembly.GetName().Version;
+        static readonly Version ENGINE_VERSION = typeof(ExtensionService).Assembly.GetName().Version;
 
         private readonly List<ExtensionPoint> _extensionPoints = new List<ExtensionPoint>();
         private readonly Dictionary<string, ExtensionPoint> _pathIndex = new Dictionary<string, ExtensionPoint>();
 
         private readonly List<ExtensionNode> _extensions = new List<ExtensionNode>();
         private readonly List<ExtensionAssembly> _assemblies = new List<ExtensionAssembly>();
 
+        public IList<Assembly> RootAssemblies { get; } = new List<Assembly>();
```

Comment:
Odd... it was used. Looking just at the code, it appears that extension points in the `nunit.engine.core` and `nunit.engine.api` will be found but not those in `nunit.engine`. But we know they are found because the tests pass!

Obviously, in the course of implementing this three times, I must have changed the approach. I'll verify and possibly remove this.

Reply:
OK... I realized at some point that all our extension points are __currently__ defined in `nunit.engine.api` and `nunit.engine.core` (only one in the latter). I think `RootAssemblies` was needed in a three-way split, but not in a two-way split.

I think some way to specify roots to be searched for extension points will be needed at some point - for example if we want to allow extension points in nunit3-console. However, it could be removed now or just commented as being for future use. I'm OK either way.